shared/common/build/tools/generate_headers.sh

```diff
@@ -60,6 +60,11 @@ wrapper_directory=$(dirname $OPT_OUTPUT)
 
 [[ -d $wrapper_directory ]] || mkdir -p $wrapper_directory
```

Comment:
You made the same fix to generate_headers.sh in the iOS repo. Since the iOS repo is a submodule, should we simply invoke the generate_headers.sh file from there?

Reply:
I was wondering why there were two... should I update the other frameworks to use it too and delete generate_headers from this repo?

Reply:
I believe they are identical so we should be able to delete the duplicate.
I can't remember if there was a reason for the duplication (maybe a scenario where we would somehow build from a shallow clone - without submodules - with the dependencies somehow there). 
I would say, let's delete it.